docs/rules/prefer-optional-props-using-with-defaults.md

```diff
@@ -0,0 +1,61 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/prefer-optional-props-using-with-defaults
```

Comment:
Maybe better rename the rule to `vue/no-required-prop-with-default`?

Reply:
thank you! I have improved name to that